modules/social_features/social_featured_content/templates/group--featured.html.twig

```diff
@@ -51,7 +54,8 @@
   <small class="text-muted">
 
     <div class="teaser__content-line">
-      <svg class="teaser__content-type-icon" aria-hidden="true" title="{% trans %}Group type{% endtrans %}">
+      <svg class="teaser__content-type-icon">
```

Comment:
This SVG should be hidden because there's an sr-only line with the same information right after it.

Reply:
Fixed